<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="io_method" value="mmap"/>
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="framerate" value="45" />
    <param name="pixel_format" value="yuyv" />
    <param name="autofocus" value="false" />
    <param name="focus" value="-1" />
    <param name="autoexposure" value="true" />
    <param name="exposure" value="100" />
    <param name="auto_white_balance" value="true" />
    <param name="white_balance" value="4000" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="camera_name" value="usb_cam" />
    <param name="camera_info_url" value="file://$(find usb_cam)/calibration/ost.yaml" />
  </node>
  <node name="image_proc" pkg="image_proc" type="image_proc" ns="usb_cam"/>

  <node name="image_view" pkg="image_view" type="image_view" respawn ="true" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true"/>
  </node>
  <node name="image_view_rect_color" pkg="image_view" type="image_view" respawn="true" output="screen">
    <remap from="image" to="/usb_cam/image_rect_color"/>
    <param name="autosize" value="true" />
  </node>

</launch>
